src/controllers/HighLightedText.js

```diff
@@ -0,0 +1,55 @@
+import models from '../db/models';
+import errorResponse from '../helpers';
+
+const { HighlightedText: HighLightedTextModel, Article } = models;
+/**
+ * @description class representing comment on Highlightedtext Controller
+ * @class HighLightedText
+ */
+class HighLightedText {
+  /**
+   * @description - This method is responsible for creating new comment on a highlighted text.
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
+   * @memberof HighLightedText
+   */
+  static async createComment(request, response) {
+    const articleSlug = request.params.slug;
+    const userId = request.userData.payload.id;
+    const { text, comment } = request.body;
+    try {
+      const article = await Article.findOne({
+        where: {
+          slug: articleSlug
```

Comment:
Hi Emeka,  nice work here, but I think you could have reduced the noise here by destructuring the 'articleSlug' in line 19. You could have done something like...

const { slug } = request.params.slug;

Then line 25 becomes, ...where: { slug }.

I would like to know what you think, though.

Reply:
That's true. It has been fixed.